src/test/java/com/google/sps/TripServletTest.java

```diff
@@ -92,6 +110,67 @@ public void tearDown() {
   }
 
   @Test
+  public void testPutTripDayInDatastore() throws Exception {
+    // set mock object behavior
+    when(request.getParameter("inputDestination")).thenReturn(INPUT_DESTINATION);
+
+    // create key
+    long test = 123;
```

Comment:
since you aren't using this variable again, I think it's fine to just write "KeyFactory.createKey("test", 123)"